test/functional/services/test_subjects.js

```diff
@@ -13,7 +13,7 @@ export function TestSubjectsProvider({ getService }) {
   const defaultFindTimeout = config.get('timeouts.find');
```

Comment:
Is this used anywhere? (sorry I didn't do a search, just asking.)

Reply:
i mean is `defaultFindTimeout` used anywhere.

Reply:
Yes, in the `click` method.